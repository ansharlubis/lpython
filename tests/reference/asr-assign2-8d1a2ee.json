{
    "basename": "asr-assign2-8d1a2ee",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/assign2.py",
    "infile_hash": "29a0112ddad4d12c7dc66c21cc5d668f1d700bc79158ca6792dd240a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-assign2-8d1a2ee.stdout",
    "stdout_hash": "d8631341fe7a78fe729132b4fc8b5c825f18fd970c29dbef2a99fa8a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}