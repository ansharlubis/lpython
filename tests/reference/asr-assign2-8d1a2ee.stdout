([1m[35mTranslationUnit[39m[0m 
   ([33mSymbolTable[39m 
      1 
      {
         f: 
            ([1m[35mVariable[39m[0m 
               1 
               f [1m[32m
               Local[39m[0m 
               ([1m[35mCast[39m[0m 
                  ([1m[35mRealConstant[39m[0m   1.23456788999999989e+00 
                     ([1m[35mReal[39m[0m 8 [])) [1m[32m
                  RealToReal[39m[0m 
                  ([1m[35mReal[39m[0m 4 []) 
                  ([1m[35mRealConstant[39m[0m   1.23456788999999989e+00 
                     ([1m[35mReal[39m[0m 4 []))) () [1m[32m
               Default[39m[0m 
               ([1m[35mReal[39m[0m 4 []) [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Required[39m[0m .false.), 
         f2: 
            ([1m[35mVariable[39m[0m 
               1 
               f2 [1m[32m
               Local[39m[0m 
               ([1m[35mRealConstant[39m[0m   1.23456789012340007e+00 
                  ([1m[35mReal[39m[0m 8 [])) () [1m[32m
               Default[39m[0m 
               ([1m[35mReal[39m[0m 8 []) [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Required[39m[0m .false.), 
         i: 
            ([1m[35mVariable[39m[0m 
               1 
               i [1m[32m
               Local[39m[0m ([1m[35mIntegerConstant[39m[0m [36m5[39m 
               ([1m[35mInteger[39m[0m 4 [])) () [1m[32m
               Default[39m[0m 
               ([1m[35mInteger[39m[0m 4 []) [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Required[39m[0m .false.), 
         i2: 
            ([1m[35mVariable[39m[0m 
               1 
               i2 [1m[32m
               Local[39m[0m 
               ([1m[35mCast[39m[0m ([1m[35mIntegerConstant[39m[0m [36m53430903434[39m 
                  ([1m[35mInteger[39m[0m 4 [])) [1m[32m
                  IntegerToInteger[39m[0m 
                  ([1m[35mInteger[39m[0m 8 []) ()) () [1m[32m
               Default[39m[0m 
               ([1m[35mInteger[39m[0m 8 []) [1m[32m
               Source[39m[0m [1m[32m
               Public[39m[0m [1m[32m
               Required[39m[0m .false.)
      }) 
   [])
(TranslationUnit (SymbolTable 1 {f: (Variable 1 f Local (Cast (RealConstant   1.23456788999999989e+00 (Real 8 [])) RealToReal (Real 4 []) (RealConstant   1.23456788999999989e+00 (Real 4 []))) () Default (Real 4 []) Source Public Required .false.), f2: (Variable 1 f2 Local (RealConstant   1.23456789012340007e+00 (Real 8 [])) () Default (Real 8 []) Source Public Required .false.), i: (Variable 1 i Local (IntegerConstant 5 (Integer 4 [])) () Default (Integer 4 []) Source Public Required .false.), i2: (Variable 1 i2 Local (Cast (IntegerConstant 53430903434 (Integer 4 [])) IntegerToInteger (Integer 8 []) ()) () Default (Integer 8 []) Source Public Required .false.), main_program: (Program (SymbolTable 2 {}) main_program [] [])}) [])
